src/biogeophys/TemperatureType.F90

```diff
@@ -609,17 +609,17 @@ subroutine InitHistory(this, bounds, is_simple_buildtemp, is_prog_buildtemp )
             avgflag='A', long_name='Growing degree days base 10C from planting', &
             ptr_patch=this%gdd10_patch, default='inactive')
 
-       this%gdd020_patch(begp:endp) = spval
+       this%gdd020_patch(begp:endp) = nan
```

Comment:
We normally initialize history fields to spval so that grid cell averages will come out correctly for fields that don't have valid values over some patches - e.g., non-vegetated (or non-crop) patches in this case. Can you comment on why you made this change? And have you confirmed that it works to output grid cell averages of these fields with this change to NaN?

Reply:
Good point! I think I was trying to stick as closely as possible to how things were done for some other variable that used `accumulMod`, but of course the _history_ field is different from the _accumulation_ field, so I don't need to worry about matching the former. Will change back to `spval`.